kratos/input_output/stl_io.cpp

```diff
@@ -52,6 +80,33 @@ void StlIO::ReadModelPart(ModelPart & rThisModelPart)
         ReadSolid(rThisModelPart);
 }
 
+void StlIO::WriteModelPart(ModelPart & rThisModelPart)
+{    
+    (*mpInputStream) << "solid compensated_skin_part\n";
```

Comment:
please don't hard code the name; you could pass it as a default string for example